exporter/ibft/reader.go

```diff
@@ -105,53 +110,53 @@ func (r *reader) validateDecidedMsg(msg *proto.SignedMessage) error {
 	return p.Run(msg)
 }
 
-// processDecidedMessage is responsible for processing an incoming decided message.
-// If the decided message is known or belong to the current executing instance, do nothing.
-// Else perform a sync operation
-func (r *reader) processDecidedMessage(msg *proto.SignedMessage) error {
-	r.logger.Debug("processing a valid decided message", zap.Uint64("seq number", msg.Message.SeqNumber), zap.Uint64s("signer ids", msg.SignerIds))
-
-	// if we already have this in storage, pass
-	known, err := r.decidedMsgKnown(msg)
-	if err != nil {
-		return errors.Wrap(err, "failed to check if decided msg is known")
-	}
-	if known {
-		return nil
-	}
-
-	shouldSync, err := r.decidedRequiresSync(msg)
-	if err != nil {
-		return errors.Wrap(err, "failed to check if decided sync is required")
-	}
-	if shouldSync {
-		r.logger.Warn("[not implemented yet] should sync validator data",
-			zap.String("lambda", hex.EncodeToString(msg.Message.Lambda)))
-	}
-	return nil
-}
-
-func (r *reader) decidedMsgKnown(msg *proto.SignedMessage) (bool, error) {
-	found, err := r.storage.GetDecided(msg.Message.Lambda, msg.Message.SeqNumber)
-	if err != nil && err.Error() != kv.EntryNotFoundError {
-		return false, errors.Wrap(err, "could not get decided instance from storage")
-	}
-	return found != nil, nil
-}
-
-// decidedRequiresSync returns true if:
-// 		- highest known seq lower than msg seq
-// 		- AND msg is not for current instance
-func (r *reader) decidedRequiresSync(msg *proto.SignedMessage) (bool, error) {
-	if msg.Message.SeqNumber == 0 {
-		return false, nil
-	}
-	highest, err := r.storage.GetHighestDecidedInstance(msg.Message.Lambda)
-	if err != nil {
-		if err.Error() == kv.EntryNotFoundError {
-			return msg.Message.SeqNumber > 0, nil
-		}
-		return false, errors.Wrap(err, "could not get highest decided instance from storage")
-	}
-	return highest.Message.SeqNumber < msg.Message.SeqNumber, nil
-}
+//// processDecidedMessage is responsible for processing an incoming decided message.
```

Comment:
@amir-blox should this be commented out?

Reply:
It was removed due to irrelevancy